Source/WebInspectorUI/UserInterface/Views/OverrideUserPreferencesPopover.js

```diff
@@ -145,11 +173,17 @@ WI.OverrideUserPreferencesPopover = class OverrideUserPreferencesPopover extends
         switch (preferenceValue) {
         case WI.CSSManager.UserPreferenceDefaultValue:
             return WI.UIString("System (%s)", "System @ User Preferences Overrides", "Label for a preference that matches the default system value. The system value is shown in parentheses.").format(this._userPreferenceValueLabel(systemValue));
-        case InspectorBackend.Enum.Page.UserPreferenceValue.Reduce:
-        case InspectorBackend.Enum.Page.UserPreferenceValue.More:
+        case InspectorBackend.Enum.Page.UserPreferenceValue?.Reduce:
```

Comment:
Probably fine, but should we also assert the `preferenceValue` isn't `null`/`undefined` before the switch-case statement now that many of these cases could be checking for `null` on older remote ends without `InspectorBackend.Enum.Page.UserPreferenceValue`

Reply:
I don't think I understand. There already is an assert for unknown preferences after the switch statement.

All code paths where `_userPreferenceValueLabel()` is called are guarded by the existence of a user preference name:
- `_defaultUserPreferencesDidChange()` iterates over `WI.cssManager.defaultUserPreferences` that are set in response to the event from the backend with supported default preference names.
- `_createSelectElement()` is guarded by the existence of a specific preference name.